DataFormats/Histograms/src/classes_def.xml

```diff
@@ -73,4 +73,10 @@
 
  <class name="DQMToken" ClassVersion="0"/>
  <class name="edm::Wrapper<DQMToken>" persistent="false"/>
+ <class name="MonitorElementCollection" persistent="true"/>
+ <class name="edm::Wrapper<MonitorElementCollection>" persistent="true"/>
+ <class name="MonitorElementData" persistent="true"/>
+ <class name="edm::Wrapper<MonitorElementData>" persistent="true"/>
+ <class name="vector<const MonitorElementData*>" persistent="true"/>
```

Comment:
Shouldn't this be `<class name="vector<std::unique_ptr<const MonitorElementData>>" >` ?

Reply:
Indeed, I changed the actual `MECollection` a lot for this PR but did not update any of this. Will have a look tomorrow.